src/main/java/mate/academy/Main.java

```diff
@@ -39,17 +50,48 @@ public static void main(String[] args) {
         tomorrowMovieSession.setMovie(fastAndFurious);
         tomorrowMovieSession.setShowTime(LocalDateTime.now().plusDays(1L));
 
+        MovieSession plusTwoDaysMovieSession = new MovieSession();
+        plusTwoDaysMovieSession.setCinemaHall(secondCinemaHall);
+        plusTwoDaysMovieSession.setMovie(fastAndFurious);
+        plusTwoDaysMovieSession.setShowTime(LocalDateTime.now().plusDays(2L));
+
         MovieSession yesterdayMovieSession = new MovieSession();
         yesterdayMovieSession.setCinemaHall(firstCinemaHall);
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
+        movieSessionService.add(plusTwoDaysMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService =
+                (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        try {
+            authenticationService.register("121212@email.com", "121212");
+        } catch (RegistrationException e) {
+            throw new RuntimeException(e);
+        }
+        User user;
+        try {
+            user = authenticationService.login("121212@email.com", "121212");
+        } catch (AuthenticationException e) {
+            throw new RuntimeException(e);
```

Comment:
Catching `AuthenticationException` and rethrowing it as a `RuntimeException` without additional context might not provide useful information for debugging. Consider logging the exception or providing a more descriptive message.